### DISCUSSION TITLE
Error while compiling in Visual Studio 2015

### DISCUSSION THREAD

**Discussion by thanhcong2k7** [olsh/curl-to-csharp#39] [General]
When I copy the code converted from website, paste it into my code and compile, it showed up error like this:
`
CS4033 : The 'await' operator can only be used within an async method. Consider marking this method with the 'async' modifier and changing its return type to 'Task'.
`
Code:
```
using (var httpClient = new HttpClient())
                {
                    using (var request = new HttpRequestMessage(new HttpMethod("POST"), "http://localhost/"))
                    {
                        var multipartContent = new MultipartFormDataContent();
                        multipartContent.Add(new StringContent(user + " " + timehour + timeday + " " + chatBox.Text),"content");
                        request.Content = multipartContent;

                        var response = await httpClient.SendAsync(request); //Error here
                    }
                }
```

(Sorry if my English was bad :< )

**Comment by olsh** [comment]
Hi @thanhcong2k7,

Yes, the converter generates code that uses C# v5 features like `async`/`await`. 
So you should migrate your project to .NET Framework 4.5 or newer. Visual Studio 2015 supports .NET Framework 4.5.
